doc/nrf/protocols/wifi/raw_tx_operation.rst

```diff
@@ -25,23 +25,28 @@ To use the raw packet transmit feature in your applications, you must enable the
 TX injection mode
 *****************
 
-The nRF70 Series device supports raw 802.11 packet transmission when TX injection mode is enabled in the nRF Wi-Fi® driver.
-To enable the TX injection mode of operation, you can use the ``NET_REQUEST_WIFI_MODE`` network management runtime API.
-This API can be used to disable TX injection mode in the driver when raw 802.11 packet transmission is not required by the application.
+The nRF70 Series device supports raw 802.11 packet transmission when TX injection mode is enabled in the nRF Wi-Fi driver.
 
-TX injection mode can be enabled when the nRF70 Series device operates in Station mode, both when it is operating in a connected Station mode or in a non-connected Station mode (not connected to an Access Point (AP)).
+TX injection mode can be enabled when the nRF70 Series device primary mode of operation is set to Station mode or Monitor mode.
+When the nRF70 Series device is configured in Station mode, TX injection mode can be enabled both when the device is either connected to an Access Point (AP) or not connected to one.
```

Comment:
Please check that this suggestion still maintained the information you intended.
```suggestion
When the nRF70 Series device is configured in Station mode, TX injection mode can be enabled regardless of whether the device is connected to an Access Point (AP) or not.
```